### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by lewiuberg**
Add citation file to the repository to enable GitHub functionality.

Skjermbilde 2021-12-19 kl  11 39 04

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - "Accuracy"
+preferred-citation:
+  type: article
+  authors:
+  - family-names: "Haghighi"
+    given-names: "Sepand"
+  - family-names: "Zolanvari"
+    given-names: "Alireza"
+  - family-names: "Sabouri"
+    given-names: "Sadra"
```
Comment (sepandhaghighi) [inline CITATION.cff:30]: @lewiuberg 
The author's section for this paper is not correct

Authors list :
```
  - family-names: "Haghighi"
    given-names: "Sepand"
  - family-names: "Jasemi"
    given-names: "Masoomeh"
  - family-names: "Hessabi"
    given-names: "Shaahin"
  - family-names: "Zolanvari"
    given-names: "Alireza"
```

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@lewiuberg Thanks for your efforts 💯 
Please consider the following:

1. Take a look at my comment about the article authors
2. Update `AUTHORS.md`
	- Add your name under `# Other Contributors #` section

**Comment by lewiuberg** [comment]
Thanks for the Feedback! I'll fix 👍🏻

**Comment by lewiuberg** [comment]
Sorry! I didn't see that there where an extra author in the article compared with the repo. I included the authors' ORCID as well. Thanks for this great package, it will be quoted in my bachelor thesis 😊👍

**Review by sepandhaghighi — APPROVED** [review]

**Pull request merged**